### ISSUE TITLE
Works with Datomic Ions?

### ISSUE THREAD

**Issue by tslocke** [ptaoussanis/sente#354]
With apologies for opening an issue to ask a question, is there any reason sente would/wouldn't work in a Datomic Ions server environment? The complication is that with Ions, browsers can only reach the server via AWS API Gateway.

In my case the app is running a regular ring stack (based off the Luminus template).

**Comment by usernms** [comment]
Hey @tslocke, do you remember where this led? I am similarly curious, and was scouring the web to see if I could save myself some time before trying to get WebSockets working with Ions (either Lambda proxies or HTTP-Direct).

**Comment by tslocke** [comment]
I didn’t end up going further with it.

**Comment by hadils** [comment]
Hi @usernolan Did you make any progress with this? I have the exact use case and I'm trying to decide whether or on to use `sente`

**Comment by usernolan** [comment]
Hi @hadils! I played around with it, and it ended up putting up a bigger barrier I was willing to endure. I didn't save any of the experimental code unfortunately. Most of it was messing around with trying to configure the provisioned lambdas in our provisioning tool.

Where I think I landed was that it would probably be easier to publish a regular Datomic Ion and invoke it from a handler that was part of an API Gateway specifically configured to use WebSockets vs. REST, as in the [websockets example](https://github.com/aws-samples/simple-websockets-chat-app/blob/master/template.yaml#L24).

TL;DR I never got it wired up correctly, but I'm sure it's possible. Let me know if you get it working!

**Comment by ptaoussanis** [comment]
Closing for inactivity as part of issue triage.
Please feel free to re-open if this issue is still relevant.
